.github/workflows/audit.yml

```diff
@@ -0,0 +1,15 @@
+name: Audit-Check
+on:
+  push:
+    paths:
+      - "**/Cargo.toml"
+      - "**/Cargo.lock"
```

Comment:
This could be run as a cron job / e.g. scheduled nightly. See: https://help.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events-schedule

I'm wondering if this shouldn't be consistent for informal repos? see https://github.com/informalsystems/ibc-rs/blob/db92c7241ae6ea723bb4fbe1148de71f4f8a9493/.github/workflows/audit.yaml#L8-L9

Reply:
You can do both: https://github.com/iqlusioninc/armistice/blob/develop/.github/workflows/security_audit.yml#L8

If you cache the install of `cargo audit` it doesn't take that long, so might as well (I don't block merging on it though, since it only runs with `Cargo.*` changes

Reply:
will add cache and cron job like in the armstice repo. will replicate in the ibc repo as well